src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
In the endpoint we handle successful responses.*
We can also handle endpoint specific errors. 

Then in the GenericResource class we handle generic stuff. 


\* The reason why we should do it this way will be more clear when/if we decide to hydrate the response into domain objects. Ie returning a UserInfo object instead of a plain array. Here is a quick skiss of how these domain object might look (by @magnusnordlander)

![image](https://user-images.githubusercontent.com/1275206/28873888-be966dfc-778f-11e7-9457-a76467b6ce1c.png)


![](https://files.slack.com/files-pri/T04481ESP-F69T0E65Q/skiss_auth0.png)

![](https://files.slack.com/files-pri/T04481ESP-F69T0E65Q/skiss_auth0.png)